ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/governedParticipantsServiceAgent.service.ts

```diff
@@ -18,8 +18,11 @@ export class GovernedParticipantsServiceAgent extends OperatorServiceAgent<any>
         super(session, configuration, http, logger);
     }
 
-    public add(alias: string): Observable<void> {
-        return this.postObservable(`/participants`, { alias });
+    public addParticipant(
+      studyGuid: string,
+      body: Record<string, any> = {},
+    ): Observable<HttpResponse<{ ddpUserGuid: string }>> {
+        return this.postObservable(`/studies/${studyGuid}/participants`, body);
```

Comment:
Where can we find the backend changes for these new services?

Reply:
Here: https://github.com/broadinstitute/ddp-study-server/pull/463
As @fat23cat suggested I'd wait for backend changes to be approved first.